js/widgets/status.js

```diff
@@ -185,16 +187,16 @@ function StatusMatrix() {
         widgetWindow.sendToCenter();
     };
 
-    this.updateAll = () => {
+    updateAll = () => {
```

Comment:
Please use `update() {` syntax to maintain code style consistency across the project.